packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswerContent/templates/generatedMarkdownContent.css

```diff
@@ -74,6 +74,10 @@ code {
   border-radius: var(--quantic-genqa-border-radius);
   text-wrap: nowrap;
```

Comment:
I see this, wouldn't removing this line solve the issue?

In both case we should clean things a bit here, we can't have css rules the tries to apply a `nowrap` and in the same time other rules that tries to apply a `pre-wrap`

Reply:
it "works" but doesn't display it super nicely for users:

<img width="295" alt="image" src="https://github.com/user-attachments/assets/315b946b-099c-4fb9-949e-c1cb510bf82b">

I think the proposed solution ensures the code block is more readable. 
I removed the extra property, forgot to remove it :)

Reply:
Well the main thing causing the difference is the `display: block` which makes the `code` takes a whole new line, instead of being an inline element, 

it makes this one more readable, but not all the time, for example in the same answer it will cause this:
<img width="514" alt="Screenshot 2024-08-08 at 12 07 30 PM" src="https://github.com/user-attachments/assets/ca61fc66-d3a1-4919-8467-3cc64b3a2f7b">

instead of this:
<img width="526" alt="Screenshot 2024-08-08 at 12 08 04 PM" src="https://github.com/user-attachments/assets/f664f342-923d-4d54-9143-64d7ca757b7d">

So playing with the display from `inline` to `block` is not always good and may cause other issues.

Reply:
Yeah, thats a fair point. I looked into it and I could just leave the `display: inline`. That will "fix" the issue here, but probably make a lot of the code generated look not ideal like the example at the top. I suspect that cases like the one at the top is probably a majority of cases here but I could be wrong.

We could also have a selector like `ul li code { display: inline; }` for code snippets that are part of lists like you raised up. I tried to think of other situations where the code elements would look weird with the `display: block` but its kinda hard to blindly test them...

This solution for lists seems to work well:
<img width="591" alt="image" src="https://github.com/user-attachments/assets/32e16df5-2a2b-4420-9465-df0b8652bd6a">

But as I said maybe the base `display: block` might make other stuff look weird. Also, I am aware that if we cather for this edge case (code elements in lists) we might open a rabbit hole of other edgecases. 

What do you think? Should we keep it simple and safe or aim to make code snippets look good at the risk of making other stuff a edge case?

unless @erocheleau has queries that would trigger other types of code snippets?

Reply:
I think we should keep it simple, the problem is that the code was overflowing, so I think a simple fix of just removing `textwrap: nowrap` would do this for us.

Reply:
@erocheleau That's the discussion @SimonMilord  mentioned in the scrum, let us know if you have more context on that since you worked on this component.

Reply:
No more context than what was already said, we don't have any report or nothing of examples of issues caused by this, so let's just fix the no-wrap for now and that's all.